scalardl-node-client-sdk.js

```diff
@@ -17,32 +17,83 @@ class ClientService extends ClientServiceBase {
    */
   constructor(properties) {
     const ledgerClientUrl =
-      `${properties['scalar.dl.client.server.host']}:` +
-      `${properties['scalar.dl.client.server.port']}`;
+        `${properties['scalar.dl.client.server.host']}:` +
+        `${properties['scalar.dl.client.server.port']}`;
```

Comment:
I'm not sure why google javascript will add indentation here, is it the same for you as well Plenty?

Reply:
I guess it is made by WebStorm.
Google JavaScript style guide will do indent with 2 spaces
https://google.github.io/styleguide/jsguide.html#formatting-block-indentation